ms/spaceness.tex

```diff
@@ -281,6 +286,26 @@ \subsection{Demographic Parameters}
 All calculations were performed with information recorded in the tree sequence,
 using pyslim \citep{pyslim}.
 
```

Comment:
Here's my discussion of dispersal, and computatoin of N_W with effective quantities!